src/connectors/faiss_file.py

```
@@ -1,17 +1,123 @@
+        doc = DocxDocument(io.BytesIO(data))
+        paragraphs = [para.text for para in doc.paragraphs if para.text.strip()]
+        tables_text = []
+        for table in doc.tables:
+            for row in table.rows:
+                row_text = [
+                    cell.text.strip() for cell in row.cells if cell.text.strip()
+                ]
+                if row_text:
+                    tables_text.append(" | ".join(row_text))
+        return "\n".join(paragraphs + tables_text).strip()
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)